src/controllers/organizations.js

```diff
@@ -195,8 +195,32 @@ async function updateOrgProfile(req, res) {
     }
 }
 
+async function deleteSubscription(req, res) {
```

Comment:
Please replace this with `unFollowOrga` to be more consistent with the other function you created.